app/src/test/java/pm/gnosis/heimdall/ui/safe/SafeOverviewViewModelTest.kt

```diff
@@ -228,40 +228,148 @@ class SafeOverviewViewModelTest {
         val testObserver = TestObserver<Result<Boolean>>()
         val account = Account(BigInteger.ZERO)
         val balance = LOW_BALANCE_THRESHOLD
+        val subject = PublishSubject.create<Wei>()
```

Comment:
I think this subject is messing up the test for the repeatWhen.

`Observable.just(balance)` should be enought and it should be called agains once you call `it.advanceTimeBy(BALANCE_CHECK_TIME_INTERVAL_SECONDS, TimeUnit.SECONDS)`